dropwizard-parent/pom.xml

```diff
@@ -36,11 +36,43 @@
                 <jdk>[9,)</jdk>
             </activation>
             <properties>
-                <maven.compiler.release>8</maven.compiler.release>
-                <maven.compiler.source>8</maven.compiler.source>
-                <maven.compiler.target>8</maven.compiler.target>
```

Comment:
Source and target properties already set above. Moved release setting into the configuration for maven-compiler-plugin.